feature/session/src/main/java/io/github/droidkaigi/confsched2019/session/ui/widget/SessionsItemDecoration.kt

```diff
@@ -42,16 +46,24 @@ class SessionsItemDecoration(
         }
     }
 
+    // Keep SparseArray instance on property to avoid object creation in every onDrawOver()
+    private val positionToViews = SparseArray<View>()
```

Comment:
`adapterPositionToViews` is preferable rather than `positionToViews` in my opinion. What do you think about it?

Reply:
It seems other `private val` fields are above `public val`. Could you please sort fields? 👀

Reply:
Included both suggestion at 49e58360076432e53ecdbe3c73ea12f9a77e780c.